@navikt/core/react/src/form/combobox/Combobox.tsx

```diff
@@ -0,0 +1,247 @@
+import cl from "clsx";
+import React, {
+  forwardRef,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+} from "react";
+import { BodyShort, Label, mergeRefs } from "../..";
+import ClearButton from "./ClearButton";
+import FilteredOptions from "./FilteredOptions/FilteredOptions";
+import { useFilteredOptionsContext } from "./FilteredOptions/filteredOptionsContext";
+import SelectedOptions from "./SelectedOptions/SelectedOptions";
+import ToggleListButton from "./ToggleListButton";
+import { ComboboxClearEvent, ComboboxProps } from "./types";
+import { useCustomOptionsContext } from "./customOptionsContext";
+import { useSelectedOptionsContext } from "./SelectedOptions/selectedOptionsContext";
+import ComboboxWrapper from "./ComboboxWrapper";
+import { useInputContext } from "./Input/inputContext";
+import Input from "./Input/Input";
+
```

Comment:
Kan legge til JSdoc her (se andre komponenter)

Reply:
Kan være vi bør se litt på strukturen vår her (det vi eksponerer som Combobox er egentlig ComboboxProvider).
Men har lagt til JSDoc i types.ts, men skal få lagt inn noe for selve komponenten også.

Reply:
Har skrevet en liten JSDoc for komponenten (i ComboboxProvider, som er den vi eksporterer) med kodeeksempel.